exp/production.md

```diff
@@ -0,0 +1,84 @@
+When you are planning a production deployment, you should consider the aspects that you want to customize for best results. This topic covers the most common and important requirements when you are planning to move your Anbox Cloud deployment to production. While there could be other specifically tailored needs, this topic helps you identify your basic requirements and deployment strategy.
+
+## Hardware requirements 
+
+While you can run Anbox Cloud on a single machine, you should use several machines for a production environment. Depending on your workload and the type of deployment you choose, hardware requirements can differ. See [requirements](https://anbox-cloud.io/docs/requirements) to get an idea about the kind of hardware that is required to run a full Anbox Cloud deployment with the streaming stack or a minimal core version of the deployment without the streaming stack. 
```

Comment:
You will need several machines for a production deployment in any case. We do not consider the appliance production ready.

Reply:
I think also we need to differentiate the term "hardware" and "cloud resources". "hardware" points more towards physical hardware but in most cases our users use a public cloud.